diff_drive_controller/CMakeLists.txt

```diff
@@ -30,6 +30,12 @@ foreach(Dependency IN ITEMS ${THIS_PACKAGE_INCLUDE_DEPENDS})
   find_package(${Dependency} REQUIRED)
 endforeach()
 
+find_package(generate_parameter_library REQUIRED)
+
+generate_parameter_library(diff_drive_controller_parameters
+  src/parameters.yaml
```

Comment:
Please rename this to `force_torque_sensor_broadcaster_parameters.yaml`.

Having many files in the repo with the name `parameters.yaml` will make searching for the correct file annoying.

Reply:
I think you meant `diff_drive_controller_parameter.yaml`. I will make the chnage